src/main.cpp

```diff
@@ -25,59 +25,62 @@
 
 #include "vcf/file_structure.hpp"
 #include "vcf/validator.hpp"
+#include "vcf/report_writer.hpp"
 
+// TODO any reason why this namespace is indented 4 spaces?
```

Comment:
I would go with coherent indentation, but those changes are a pain for reviewing... I guess it is not the end of the world this way